Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string Name1 = "todd";
```

Comment:
Variables should be camelCased named (first letter non capital followed by capitalized letter of every next word).